gulpfile.js

```diff
@@ -1,34 +1,38 @@
-var gulp = require('gulp');
-var gutil = require('gulp-util');
-var webpack = require('webpack');
+const gulp = require('gulp');
+const log = require('fancy-log');
+const webpack = require('webpack');
 
-
-gulp.task('default', ['webpack', 'watch']);
-gulp.task('dev', ['default']);
-gulp.task('build', ['set-production', 'webpack']);
-
-gulp.task('set-production', [], function () {
+const setProduction = function (cb) {
   process.env.NODE_ENV = 'production';
-});
+  if (cb) {
+      cb();
+  }
+};
 
-var webpackOnBuild = function (done) {
+const webpackOnBuild = function (done) {
```

Comment:
This could be written as an arrow function, `done => (err, stats) => {` etc

Reply:
I was hesitant to do that because of functional context (this-keyword)- It is handled different between function and arrow function. I did not want to have to worry about that along with the many other things I had to worry about.

Reply:
OK! I was comparing your two npm package updates and noticed that you made this change in the other one: https://github.com/ENCODE-DCC/encoded/pull/2913/commits/9ac80faf5c5979a3bca21eb6cb3b8f80e113343d#diff-b9e12334e9eafd8341a6107dd98510c9L14

Reply:
Since the code works and the change is not significant, I left it as-is. It can be updated in a future release.